paws.common/R/config.R

```diff
@@ -150,29 +150,16 @@ get_env <- function(variable) {
   return("")
 }
 
-# Get the value of an OS environment variable.
-# NOTE: Does not work on Windows.
-get_os_env <- function(var) {
```

Comment:
`get_os_env()` and `get_profile_name()` moved below (alphabetical) without touching function

Reply:
happy for them to be moved